src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,15 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+
+        for (int i = 0; i < 3; i++) {
+            StringBuilder resultOf3 = new StringBuilder();
+            resultOf3.append(i + 1);
+            resultOf3.append(": ");
+            resultOf3.append(lottery.getRandomBall().getNumber());
+            resultOf3.append(lottery.getRandomBall().getColor());
+            System.out.println(resultOf3.toString());
```

Comment:
The `StringBuilder`'s `toString()` method is called implicitly when passing it to `System.out.println`, so you can omit the `toString()` call here.